components/status/StatusLink.vue

```diff
@@ -8,7 +8,8 @@ const props = defineProps<{
 
 const el = ref<HTMLElement>()
 const router = useRouter()
-const statusRoute = $computed(() => getStatusRoute(props.status))
+const isSingleInstanceServer = useRuntimeConfig().public.singleInstance
+const statusRoute = $computed(() => getStatusRoute(props.status, isSingleInstanceServer))
```

Comment:
Updated the routes helper `getStatusRoute` to accept single instance boolean.  Which should handle the `ctrl+click` use case